tests/financial/test_ice_futures_europe.py

```diff
@@ -0,0 +1,62 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/python-holidays
+#  License: MIT (see LICENSE file)
+
+from unittest import TestCase
+
+from holidays.financial.ice_futures_europe import ICEFuturesEurope, IFEU
+from tests.common import CommonFinancialTests
+
+
+class TestICEFuturesEurope(CommonFinancialTests, TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass(ICEFuturesEurope, years=range(2000, 2100))
+
+    def test_market_aliases(self):
+        self.assertAliases(ICEFuturesEurope, IFEU)
+
+    def test_no_holidays(self):
+        self.assertNoHolidays(ICEFuturesEurope(years=2013))
+
+    def test_2021(self):
+        self.assertHolidays(
+            ICEFuturesEurope(years=2021),
+            ("2021-01-01", "New Year's Day"),
+            ("2021-04-02", "Good Friday"),
+            # Saturday so not a market holiday
```

Comment:
I'm not sure if the comment is useful here. Could you explain what you meant by it? It seems a bit controversial within the `assertHolidays` context.

Thank you!

Reply:
I see, thanks for the explanation. 
I've implemented https://github.com/vacanza/python-holidays/pull/1796 today in order to improve this. Now we have `SAT_TO_NONE` rule. 

I also updated the README.

I believe it's good to go.